src/domain/app/header/__tests__/Header.test.tsx

```diff
@@ -250,19 +251,14 @@ test('should start logout process', async () => {
 });
 
 test('should route to search page', async () => {
-  const searchValue = 'search';
   const user = userEvent.setup();
   const { history } = renderComponent();
 
-  const openSearchButton = screen.getByRole('button', {
-    name: 'Etsi tapahtumia',
-  });
-  await user.click(openSearchButton);
+  const searchLink = screen.getAllByRole('button', {
+    name: 'Hae',
+  })[0];
```

Comment:
I assume that `getAllByRole` is used because there is a separate search link in the mobile menu?

Reply:
This is actually mistake on my side 🤔 Fixed: https://github.com/City-of-Helsinki/linkedcomponents-ui/pull/258/commits/1c3592072e018a96138e9ab2fd98b431ac048fff